ntbs-service/wwwroot/css/_colors.scss

```diff
@@ -1,2 +1,4 @@
 $notified-red: #7f0029;
-$draft-grey: #999999;
\ No newline at end of file
+$draft-grey: #999999;
+$lighter-draft-grey: #CCCCCC;

```

Comment:
IS it actually used by drafts? If not, let's just go with "light-grey"?

Reply:
Yeah the header also uses the same 'notified-red' so I'll remove them and just give them some colour names, something like phe-red (or maroon?) and then grey and lighter-grey?

Reply:
yup, that sounds good!
There was a doc somewhere that actually defined them...
https://aptemus-my.sharepoint.com/:b:/p/nancy/EYDZ3skOTktGkEbF9hPqw7ABBc-cjJF-Xqpo33ZO123ITQ?e=t8Vaie
![image](https://user-images.githubusercontent.com/6727787/65445796-e5636580-de2a-11e9-90cf-91faff235fa0.png)
Looks like `PHE RED` is the name

Reply:
The cool grey here is fairly off on both design and different to the base GDS component colour for the background of the footer, want me to incorporate it or not for now? Also is it helpful if I add all of these colours (and comment where to find them on this doc if that's allowed?) to the file now?